deliver/lib/deliver/options.rb

```diff
@@ -372,6 +372,10 @@ def self.available_options
                                      description: "Metadata: Localised privacy url",
                                      optional: true,
                                      is_string: false),
+        FastlaneCore::ConfigItem.new(key: :apple_tv_privacy_policy,
+                                     description: "Metadata: Localised apple tv privacy policy",
```

Comment:
Let's make it more clear that this requires the text, and not an URL:
```suggestion
                                     description: "Metadata: Localised Apple TV privacy policy text",
```